pkg/storaged/side-panel.jsx

```diff
@@ -60,7 +60,7 @@ export class SidePanel extends React.Component {
             show_all_button = (
                 <FlexItem alignSelf={{ default: 'alignSelfCenter' }}>
                     <Button variant='link'
-                            onKeyPress={ev => ev.key === "Enter" && this.setState({ collapsed: false })}
+                            onKeyDown={ev => ev.key === "Enter" && this.setState({ collapsed: false })}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19083-20230710-104935-c8068a06-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)